features/cli/policy.feature

```diff
@@ -476,19 +476,25 @@ Feature: change the policy of user/service account
   Scenario: OCP-9552:Authentication User can know which serviceaccount and SA groups can create the podspec against the current sccs by CLI
     Given I have a project
     Given I obtain test data file "authorization/scc/PodSecurityPolicyReview.json"
+    Then I wait for the steps to pass:
+    """
     Given I run the :policy_scc_review client command with:
       | f | PodSecurityPolicyReview.json |
     Then the step should succeed
     And the output should not match:
       | .*default.*restricted |
+    """
     # Test explicit serviceaccount
+    Then I wait for the steps to pass:
+    """
     Given I run the :policy_scc_review client command with:
       | serviceaccount | default                      |
       | f              | PodSecurityPolicyReview.json |
       | n              | <%= project.name %>          |
     Then the step should succeed
     And the output should not match:
       | .*default.*restricted |
+    """
```

Comment:
For "the output should **not** match", waiting is not needed. Adding the waiting time will be unnecessarily confusing. Could you remove?